integration_tests/CMakeLists.txt

```diff
@@ -831,7 +831,7 @@ RUN(NAME callback_04       IMPORT_PATH .. LABELS cpython)
 # Intrinsic Functions
 RUN(NAME intrinsics_01     LABELS cpython llvm llvm_jit NOFAST) # any
 RUN(NAME intrinsics_02     LABELS cpython llvm llvm_jit c) # floordiv
-RUN(NAME test_builtin_type LABELS cpython llvm llvm_jit c) # type
+RUN(NAME test_builtin_type LABELS cpython llvm llvm_jit) # type
```

Comment:
@kmr-srbh we do not remove any backends for existing tests (unless extremely necessary). If there is a feature not supported by any of the previously enabled backends, then we add a new/separate integration test.

Reply:
@Shaikh-Ubaid I removed the test for it due to https://github.com/lcompilers/lpython/pull/2687#issuecomment-2096218282

Reply:
Just add the set tests in a new integration test and for it enable the other supporting backends (like llvm, llvm_jit, etc.). That way, we do not stop testing any existing tests for the C (or other) backends.

Reply:
> Just add the set tests in a new integration test and for it enable the other supporting backends (like llvm, llvm_jit, etc.). That way, we do not stop testing any existing tests for the C (or other) backends.

Got it @Shaikh-Ubaid! I will remember this for future commits. :+1: